GitUtils.ps1

```diff
@@ -222,20 +222,20 @@ function Get-AliasPattern($exe) {
    "($($aliases -join '|'))"
 }
 
-function setenv($key, $value) {
-    [void][Environment]::SetEnvironmentVariable($key, $value, [EnvironmentVariableTarget]::Process)
-    Set-TempEnv $key $value
+function Set-Env($Key, $Value) {
+    Set-Item Env:$Key -Value $Value -ErrorAction SilentlyContinue
+    Set-TempEnv -Key $Key -Value $Value
 }
 
 function Get-TempEnv($key) {
     $path = Join-Path ($Env:TEMP) ".ssh\$key.env"
     if (Test-Path $path) {
         $value =  Get-Content $path
-        [void][Environment]::SetEnvironmentVariable($key, $value, [EnvironmentVariableTarget]::Process)
+        Set-Item Env:$key -Value $value -ErrorAction SilentlyContinue

```

Comment:
Why switch from `[Environment]::SetEnvironmentVariable(...)` to `Set-Item`? I don't remember why I used `SetEnvironmentVariable()` in the first place, but I'm curious what the difference is from a OpenSsh-Win32 perspective?